go/go-sql.go

```diff
@@ -23,37 +23,52 @@ import (
 	"reflect"
 	"runtime"
 	"strings"
+
+	"go.opentelemetry.io/otel/propagation"
 )
 
 const (
-  
-  
-  
-  string = "route"
-  controller string = "controller"
-  action     string = "action"
-  framework  string = "framework"
-  driver     string = "driver"
+	route       string = "route"
+	controller  string = "controller"
+	action      string = "action"
+	framework   string = "framework"
+	driver      string = "driver"
+	traceparent string = "traceparent"
 )
 
 type DB struct {
 	*sql.DB
-	options CommenterOptions
+	options         CommenterOptions
+	otelTraceparent string
```

Comment:
traceParent should not be a part of DB. It's a dynamic value that changes for every call. Anything static can be only part of `DB` struct.

We should extract traceparent from every call i.e move it to `withComment` method. As every call will be passing the same context, we can get it from the context.